ShenkinStore/Controllers/TransactionsController.cs

```diff
@@ -193,5 +165,68 @@ private bool TransactionExists(int id)
         {
             return db.Transactions.Any(e => e.TransactionId == id);
         }
+
+        // GET: Transactions/AddressAndPayment/
+        public ActionResult AddressAndPayment()
```

Comment:
Here as well this function is really big and messy, please separate to a little more understandable functions.